src/bootstrap/sanity.rs

```diff
@@ -45,7 +45,7 @@ pub fn check(build: &mut Build) {
             let target = path.join(cmd);
             let mut cmd_alt = cmd.to_os_string();
             cmd_alt.push(".exe");
-            if target.exists() ||
+            if target.is_file() ||
```

Comment:
Nah they're likely mostly historical, this was changed as I wanted it to *not* pick up directories